app/database/user.go

```diff
@@ -54,3 +56,54 @@ func (u *User) GetManagedGroupsWithCanGrantGroupAccessIds(store *DataStore) []in
 
 	return managedGroupsWithCanGrantGroupAccessIds
 }
+
+// CanWatchMembersOnParticipant checks whether the user can_watch_members on a participant
+func (u *User) CanWatchMembersOnParticipant(s *DataStore, participantID int64) bool {
```

Comment:
"CanWatchMembersOnParticipant checks whether the user can_watch_members on a participant"
This sentence does not really make sense to me, or at least is not clear.. do you mean "whether the user can watch a participant"?

Reply:
fixed